src/blas/KokkosBlas2_serial_gemv.hpp

```diff
@@ -65,16 +67,26 @@ void KOKKOS_INLINE_FUNCTION gemv(const char trans, const ScalarType& alpha,
   } else if (trans == 'C' || trans == 'c') {
     using mode = KokkosBlas::Trans::ConjTranspose;
     KokkosBlas::SerialGemv<mode, AlgoTag>::invoke(alpha, A, x, beta, y);
+    //
+    // TODO: what letter should be used here ?
+    //    * in blas "C" means conjugate-transpose
+    //    * in sparse "C" meanse conjugate and "H" conjugate-transpose...
+  } else if (trans == 'X' || trans == 'x') {
+    using mode = KokkosBlas::Trans::ConjNoTranspose;
+    KokkosBlas::SerialGemv<mode, AlgoTag>::invoke(alpha, A, x, beta, y);
```

Comment:
@lucbv @fnrizzi @kliegeois 
Would you have any suggestions for this ?

I've implemented `ConjNoTranspose` (not normally supported in BLAS/MKL) on https://github.com/kokkos/kokkos-kernels/pull/1435/commits/9741625c72c80e99b44ea22478d52a30e339573b because it was used in [BSR `spmv`](https://github.com/kokkos/kokkos-kernels/blob/b9e285f4b633bb9eeb34b4cd945b3bd747b7175f/src/sparse/impl/KokkosSparse_spmv_bsrmatrix_impl.hpp#L659-L662).

Reply:
As discussed, let's avoid `ConjNoTranspose` in the interfaces altogether (BSR `spmv` will be good calling `TeamVectorGemvInternal` with `OpConj{}` to obtain non-transposing conjugate): fixed on https://github.com/kokkos/kokkos-kernels/pull/1435/commits/2a2a84ef876dab0cb9da243fba8612508db02033.